vm/devices/net/netvsp/src/lib.rs

```diff
@@ -1193,7 +1198,7 @@ impl VmbusDevice for Nic {
             .adapter
             .num_sub_channels_opened
             .fetch_add(1, Ordering::SeqCst);
-        let r = self.insert_worker(channel_idx, open_request, state, true);
+        let r = self.insert_worker(channel_idx, open_request, state, true, false);
```

Comment:
I think at channel open time of the primary, we want `true` here--the guest has to set the filter for packets to flow. And at channel open time of the subchannel, we want to set it to whatever the primary's state is.